public/manifest.webmanifest

```diff
@@ -9,12 +9,12 @@
       "type": "image/png"
```

Comment:
Looks like the icon above () which I can't tag at is not replaced?

Reply:
The PWA icon? I thought it would be changed when the image itself was replaced?

Reply:
Ah you mean `icon-256.png`. I got a bit confused. It has been updated, but since it's the same name, it doesn't show as deleted, just changed. You can see that it is not the same by seeing ` BIN -23.5 KB (32%)`. 

Also included a screenshot of it. It is the image that is used as a favicon, so you can see that it is not the same in the Vercel preview.
<img width="966" alt="Screenshot 2020-10-02 at 17 05 09" src="https://user-images.githubusercontent.com/22390745/94938958-caaddd00-04d1-11eb-84ae-fea72220fd9c.png">

Reply:
Sorry about that one, I thought the image had been deleted 😅 
![deleted images](https://user-images.githubusercontent.com/14319313/94939971-27f65e00-04d3-11eb-9335-73a7cc45e92e.PNG)

The only difference I can see here is the percentage visible on the replaced icon. Didn't look close enough 😛